recipes/psi4/meta.yaml

```
@@ -0,0 +1,212 @@
+    - mkl-devel {{ mkl }}                    # [win]
+    - mkl                                    # [not win]
+    - mkl {{ mkl }}                          # [win]
+    - pybind11
+    - pybind11-abi
+    - python
+    # qc req'd
+    - gau2grid
+    - libint {{ libint }}
+    - libxc-c
+    - optking
+    - qcelemental
+    - qcengine
+    # qc opt'l, build-time
+    - dkh
+  run:
+    - llvm-openmp                            # [not win]
+    - intel-openmp                           # [win]
+    - libblas * *mkl
+    - mkl                                    # [not win]
+    - mkl {{ mkl }}                          # [win]
+    - msgpack-python
+    - networkx
+    - numpy
+    - python
+    - scipy
+    # qc
+    - libint {{ libint }}
+    - {{ pin_compatible('optking', upper_bound='0.5') }}
+    - {{ pin_compatible('qcelemental', upper_bound='0.50') }}
+    - {{ pin_compatible('qcengine', upper_bound='0.50') }}
+  run_constrained:
+    - pytest >=7.0.1
+    # qc opt'l, run-time
+    - pylibefp =={{ pylibefp }}
+    - dftd4-python >={{ dftd4 }}
+    - geometric >={{ geometric }}
+    - pymdi >={{ pymdi }}
+
+test:
+  imports:
+    - psi4
+  requires:
+    - pytest
+    - pytest-xdist
+    - m2w64-binutils                                                     # [win]   # provides objdump.exe
+    # qc runtime opt'l
+    - pyddx                                                              # [unix]
+    - dftd3-python                                                       # [unix]
+    - dftd4-python                                                       # [unix]
+    - toml                                                                         # needed for qcengine/dftd4-python interface
+    - pylibefp
+    - gcp-correction                                                     # [unix]
+    - geometric
+    - pymdi
+  commands:
+    - ls -l $PREFIX/lib                                                  # [unix]
+    - ls -l $PREFIX/share/cmake/psi4                                     # [unix]
+    - ls -l $SP_DIR/psi4                                                 # [unix]
+    - dir %PREFIX%\\Library\\lib                                         # [win]
+    - dir %PREFIX%\\Library\\share\\cmake\\psi4                          # [win]
+    - dir %SP_DIR%\\psi4                                                 # [win]
+    # Verify module
+    - test -f $SP_DIR/psi4/core*.so                                      # [unix]
+    - test -f $SP_DIR/psi4/driver/driver.py                              # [unix]
+    - if not exist %SP_DIR%\\psi4\\core*.pyd exit 1                      # [win]
+    - if not exist %SP_DIR%\\psi4\\driver\\driver.py exit 1              # [win]
+    # Verify executable
+    - test -f $PREFIX/bin/psi4                                           # [unix]
+    - $PREFIX/bin/psi4 --version                                         # [unix]
+    - $PREFIX/bin/psi4 --module                                          # [unix]
```

```suggestion
    - psi4 --version                                         # [unix]
    - psi4 --module                                          # [unix]
```
This should work without specifying the full path, else the installation is broken.

---

I removed the PREFIX from the windows lines, too, and they work fine.